storage/scripts/actions_tool.rs

```diff
@@ -0,0 +1,257 @@
+use bytes::{Buf, BufMut, BytesMut};
+use std::error::Error;
+use std::fs::{File, OpenOptions};
+use std::io::{BufReader, Read, Seek, SeekFrom};
+use std::path::Path;
+// use bytes::buf::BufExt;
+use std::fmt::Formatter;
+// use crate::context_action_storage::ContextAction;
+use serde::{Deserialize, Serialize};
+
+const HEADER_LEN: usize = 12;
+type Hash = Vec<u8>;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Block {
+    pub block_level: u32,
+    pub block_hash: String,
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct ActionsFileHeader {
+    pub block_height: u32,
+    pub actions_count: u32,
+    pub block_count: u32,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub enum ContextAction {
```

Comment:
This seems to be in the old format, right? I don't see any mention of `tree_hash` and `new_tree_hash`.

Reply:
Yes its for old format.